src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -797,6 +799,8 @@ private static String processPageSourceForGodMode(String content) {
                 .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}", "{*}")
                 // now (used in comments last edited date) e.g. [Thu May 07 07:52:13 UTC 2015]
                 .replaceAll(new SimpleDateFormat("EEE MMM dd ").format(now) + "[0-9]{2}:[0-9]{2}:[0-9]{2} UTC [0-9]{4}", "{*}")
+                // dynamic feedback submission numbers
+                .replaceAll("(?s)<span class=\"submissionsNumber\".*?</span>", "<span class=\"submissionsNumber\" id=\"submissionsNumber\">{*}</span>")
```

Comment:
Yes it will be faster, but perhaps speed is not as important here as the stability of god mode?

Perhaps that can break something else without us knowing, for example any developer who comes on board can display numbers or eventually we can have a page that displays numbers as such "1,234,567" and they will get replaced by god mode and the developer will be left wondering why and unless the developer knows about this specific line, they will never know what's biting them?

Also, it seems god mode is inconsistent and sometimes write things as one line and sometimes as multiple lines, the reason why we need the dot all comparison is because it is being written as multiple lines by god mode and the dot all comparison is what is able to do the multiline HTML element recognition.. without it, I haven't been able to consistently recognize the correct HTML element